frontend/src/lib/themes.ts

```diff
@@ -130,6 +131,7 @@ const darkTheme = createTheme({
     resourceToolTip: {
       color: 'rgba(255, 255, 255, 0.87)',
     },
+    notificationBorderColor: 'rgba(255,255,255,0.12)',
```

Comment:
I wonder if you checked these for contrast/vision?
(It should hopefully print out in the browser console if there's an issue, but maybe it's useful to check with the browser tools.)